lstchain/reco/dl1_to_dl2.py

```diff
@@ -491,6 +508,13 @@ def build_models(filegammas, fileprotons,
                 "The requested number of protons for the classifier training is not valid."
             ) from e
 
+    if pointing_wise_weights:
+        # Give same total weight to all events in every pointing node, by
+        # applying event-wise weights which depend on the statistics per node
+        _, _ = utils.compute_rf_event_weights(df_proton)
```

Comment:
Maybe add a comment here that `utils.compute_rf_event_weights(df_proton)` is adding the `weight` column to `df_proton`